python/py-MDAnalysis/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-MDAnalysis
+version             0.19.2
+revision            0
+categories-append   science
+platforms           darwin
+license             LGPL-2
+
+python.versions     27 36 37
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         An object-oriented toolkit to analyze molecular dynamics trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, or Amber.
+long_description    ${description}. \
+                    It also reads other formats (e.g. PDB files and XYZ format trajectories. \
+                    It can write most of the coordinate formats, too, together with atom selections for use \
+                    in Gromacs, CHARMM, VMD and PyMOL.
+
+homepage            https://www.mdanalysis.org/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  7dab230b0e9770eeb0f760310660fe26dd70d9b2 \
+                    sha256  c5395bbafa5efca2e1aee4715d26129844140c47cb8301da0293106cb969de7d \
+                    size    16457970
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-biopython \
```

Comment:
there is an issue with the use of the compiler here, at least when you follow the description [here](https://trac.macports.org/wiki/UsingTheRightCompiler#testing) it emits the warning that it's using an incorrect compiler.

The issue manifests itself with openmp where the installation scripts tries to determines if OpenMP is installed and whether the compiler supports it (```Attempting to autodetect OpenMP support... Compiler supports OpenMP```). To solve the first issue, one should probably add a variant ```openmp``` and add an explicit dependency on openmp and enable it ; otherwise it will use it in case openmp is installed, but it's not listed as a dependency so it will break when the user uninstalls openmp. The issue with not UsingTheRightCompiler is more complicated and I cannot give any more clues right now, this requires some investigation for which I don't have time right now...

Reply:
You should be able to reproduce the problem if you have ```libomp`` installed, which you might not have right now...